frontend/src/components/SetupWizard.tsx

```
@@ -0,0 +1,134 @@
+    }
+  };
+
+  return (
+    <div className="setup-wizard">
+      <div className="setup-wizard-header">
+        <span>Setup Terminal</span>
+      </div>
+      <div className="setup-wizard-body">
+        <div className="setup-wizard-form">
+          <div className="wizard-field">
+            <label>Agent Name</label>
```

Biome (2.3.14)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)